go/structs/message.go

```diff
@@ -6,11 +6,13 @@ import "time"
 type Message struct {
 	ID           string         `json:"id"`
 	Name         string         `json:"name"`
+	OriginalName string         `json:"originalName"`
```

Comment:
I don't think we should have the `originalName` as part of the message struct. I'd like it to only define the fields that are sent over the websockets.